Makefile

```diff
@@ -38,9 +40,15 @@ test-cover:
 fmt: 
 	go fmt ./...
 
+install-lint:
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v$(GOLANGCI_VERSION)
```

Comment:
will everyone have `sh` ?

Reply:
just for CI--changed the name so that's clear!